compiler-plugin/src/main/java/io/ballerina/stdlib/websub/WebSubDiagnosticCodes.java

```diff
@@ -50,7 +50,8 @@ public enum WebSubDiagnosticCodes {
             DiagnosticSeverity.ERROR),
     WEBSUB_201("WEBSUB_201",
             "Error occurred while packaging generated resources : {0}",
-            DiagnosticSeverity.ERROR);
+            DiagnosticSeverity.ERROR),
+    WEBSUB_202("WEBSUB_202", "Template generation for empty service", DiagnosticSeverity.INTERNAL);
```

Comment:
We should report this diagnostic for the code action suggestion to appear. I don't see anywhere this diagnostic is being reported. Shall we report this in the relevant place?

Reply:
Fixed with [c86c5c867a6b175087cd9eb9d1a9c47793623af4](https://github.com/ballerina-platform/module-ballerina-websub/pull/519/commits/c86c5c867a6b175087cd9eb9d1a9c47793623af4)